qiskit/scheduler/methods/basic.py

```diff
@@ -118,7 +118,7 @@ def update_times(inst_qubits: List[int], time: int = 0) -> None:
         rev_stop_times.append(stop_time)
         update_times(circ_pulse_def.qubits, stop_time)
 
-    last_stop = max(t for t in qubit_time_available.values())
+    last_stop = max(t for t in qubit_time_available.values()) if qubit_time_available else 0
```

Comment:
👍 Should we consider breaking this out and doing a stable backport?

Reply:
Ahh probably. See issue here. https://github.com/Qiskit/qiskit-terra/issues/4445